addons/explosives/ACE_Settings.hpp

```diff
@@ -7,4 +7,8 @@ class ACE_Settings {
         value = 1;
         typeName = "BOOL";
     };
+    class GVAR(ExplodeOnDefuse) {
+        value = 1;
```

Comment:
Could you include the `displayName` and `description` in here as well? This is to help server admins understand what the setting does when in the options menu for config exports.